b2sdk/sync/folder.py

```diff
@@ -294,6 +308,20 @@ def all_files(self, reporter, policies_manager=DEFAULT_SCAN_MANAGER):
             if policies_manager.should_exclude_file(file_name):
                 continue
 
+            # Do not allow relative paths in file names
+            if (file_name.startswith('..' + os.path.sep) or
```

Comment:
I think a better way to implement it would be to pass it into a regular expression containing a few `|`

Reply:
Updated as suggested.